src/repos/NotificationRepo.ts

```diff
@@ -0,0 +1,70 @@
+import * as admin from 'firebase-admin';
+import { User } from '../entities/User';
+import { Publication } from '../entities/Publication';
+import { Article } from '../entities/Article';
+import UserRepo from './UserRepo';
+import PublicationRepo from './PublicationRepo';
+import ArticleRepo from './ArticleRepo';
+
+const sendNewArticleNotification = async (
+  user: User,
+  article: Article,
+  publication: Publication,
+): Promise<void> => {
+  // Get device token of user
+  const { deviceToken } = user;
+
+  const notifTitle = publication.name;
+  const notifBody = article.title;
+
+  const message = {
+    notification: {
+      title: notifTitle,
+      body: notifBody,
+    },
+    data: {
+      userID: user.uuid,
+      articleID: article.id,
+      articleURL: article.articleURL,
+      notificationType: 'new_article',
+    },
+  };
+
+  const options = {
+    priority: 'high',
+    timeToLive: 60 * 60 * 24,
```

Comment:
what is this? do we ever need to change this so we put this in an environments or a constant?

Reply:
i don't think so, i think it probably only showed up as a change because of a linter issue